Services/Models/SimulationScript.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Diagnostics;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Exceptions;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Simulation;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Models
+{
+    public class SimulationScript
+    {
+        [JsonIgnore]
+        public string ETag { get; set; }
+        public string Id { get; set; }
+        public string Type { get; set; }
+        public string Content { get; set; }
+        public SimulationScriptPath Path { get; set; }
+        public DateTimeOffset Created { get; set; }
+        public DateTimeOffset Modified { get; set; }
+        public string Name { get; set; }
+
```

Comment:
move this line to line 20 (after Id declaration)

Reply:
Fixed.